docs/reference/2-protocols/2-device-auth.md

```diff
@@ -3,20 +3,67 @@ id: device-auth
 title: Device Authentication
 ---
 
-To access Golioth Service from a device, the device need to be properly created and provisioned.
+To access Golioth Services from a device, the device needs to be properly created and provisioned.
 
-You can manage device and project credentials using the Web Console or through the CLI.
+You can manage device level and project credentials using the Web Console or through the CLI.
 
-### Credential Types
+## CoAP Gateway address and port
 
-| Credential Type               | Description               | Supported Gateways    |
-| ----------------------------- | ------------------------- | --------------------- |
-| Per Device PSK/Pre Shared Key | Pre Shared Keys           | CoAP ✅               |
-| Per Project Certificate       | Mutual TLS Authentication | CoAP 🚧               |
+Devices can connect to the Golioth CoAP Gateway at the following endpoint:
 
-### Creating an Pre Shared Key Credential
+ - `coap.golioth.io` at UDP port `5684`, using DTLS
 
-See the getting started guide to [authorize devices](/getting-started/3-commandline/6-authorize-devices.md).
+Devices that use URL can connect using `coaps://coap.golioth.io/`.
 
-- Guides on how to use:
-  - [Using Pre Shared Keys on CoAP Gateway](/reference/protocols/coap/auth)
+The CoAP gateway requires authentication.
+
+## Authentication Methods
+
+| Authentication Method         | Description                             |
+| ----------------------------- | --------------------------------------- |
+| DTLS PSK                      | Per-device pre-shared key               |
+| DTLS Client Certificate       | Mutual certificate-based authentication |
+
+## PSK Authentication
+
+For every device, a shared secret (key) needs to be established prior to first communication.
+
+This can be as simple as generating a pre-shared key on the Golioth console, and loading it on your device.
+
+Alternatively, you can generate a key on the device, an upload it to Golioth.
+
+See the getting started guide to [authorize PSK devices](/getting-started/3-commandline/6-authorize-devices.md).
+
+
+## Client Certificate Authentication
+
+Certificate authentication is more complex to setup and use, but provides a superior level of security.
+
+It also enables you to perform zero-touch provisioning of devices. After you configure the trusted root or intermediate certificates in your Golioth project, and device connecting using a certificate signed by one of the trusted certificates will be *automatically provisioned*.
```

Comment:
> zero-touch provisioning

Might be helpful to link to a definition of this term.

Reply:
I'll leave this to the devrel team to improve upon over time. Don't know which page to link, and don't want that to block the release.